core/src/test/java/org/hibernate/ogm/backendtck/associations/manytoone/ManyToOneTest.java

```diff
@@ -42,8 +42,8 @@ public void testUnidirectionalManyToOne() throws Exception {
 		session.persist( emmanuel );
 		session.persist( jerome );
 		session.flush();
-		assertThat( getNumberOfEntities( sessions ) ).isEqualTo( 3 );
-		assertThat( getNumberOfAssociations( sessions ) ).isEqualTo( expectedAssociations() );
+		assertThat( getNumberOfEntities( session ) ).isEqualTo( 3 );
+		assertThat( getNumberOfAssociations( session ) ).isEqualTo( expectedAssociations() );
 		transaction.commit();
 		assertThat( getNumberOfEntities( sessions ) ).isEqualTo( 3 );
```

Comment:
Apart from giving me less things to review, of course, which is a very valid reason ;)